src/App.js

```diff
@@ -86,10 +86,12 @@ class PageFetchWrapper extends Component {
         return (
             <PageContext.Provider
                 value={{
-                    mode: this.state.mode
+                    mode: this.state.mode,
```

Comment:
Why don't pass the whole state here?

Reply:
![image](https://user-images.githubusercontent.com/37185433/52421629-06549180-2aba-11e9-8f32-6b47b68659ee.png)
a lot of params not needed

Reply:
I mean:

```
value={this.state}
```